apps/api/src/bootstrap.ts

```diff
@@ -1,18 +1,15 @@
-import { CONTEXT_PATH } from './config';
```

Comment:
Same goes here, I believe that the config import should be at the top most section of the file other wise the env variables from the .env won't be accisseble inside the modules. Maybe worth adding a warning comment here aswell for that line.